docker_images/setfit/requirements.txt

```diff
@@ -0,0 +1,4 @@
+starlette==0.27.0
+api-inference-community==0.0.32
+huggingface_hub==0.11.0
+setfit
```

Comment:
If possible, it's a good idea to pin a setfit version so we know exactly what's in production and we're reliable upon rebuilds